articles/azure-app-configuration/use-key-vault-references-dotnet-core.md

```diff
@@ -171,7 +171,7 @@ To add a secret to the vault, you need to take just a few additional steps. In t
 
 ## Grant your app access to Key Vault
 
-Azure App Configuration won't access your key vault. Your app will read from Key Vault directly, so you need to grant your app read access to the secrets in your key vault. This way, the secret always stays with your app. The access can be granted using either a [Key Vault access policy](../key-vault/general/assign-access-policy-portal.md) or [Azure role-based access control](../key-vault/general/rbac-guide.md).
+Azure App Configuration won't access your key vault. Your app will read from Key Vault directly, so you need to grant your app access to the secrets in your key vault. This way, the secret always stays with your app. The access can be granted using either a [Key Vault access policy](../key-vault/general/assign-access-policy-portal.md) or [Azure role-based access control](../key-vault/general/rbac-guide.md).
```

Comment:
Thank you for the update @zhiyuanliang-ms, that sounds good.

@Muksvso, is there a reason why we leave it to the reader to decide how they want to grant access (through a Key Vault access policy or using RBAC), or should we simplify this paragraph by just telling customers to grant the app the Key Vault Secrets User role?

Reply:
Both ways can work. It depends on users' preference.

Reply:
Thanks!